src/test/java/reposense/model/FormatTest.java

```diff
@@ -0,0 +1,29 @@
+package reposense.model;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class FormatTest {
+    @Test
+    public void validateFormats_alphaNumeric_success()
+            throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        Method m = Format.class.getDeclaredMethod("validateFormats", List.class);
+        m.setAccessible(true);
+        m.invoke(null, Arrays.asList("java", "7z"));
```

Comment:
Rather than using Reflection for the test (which is considered bad practice since we are breaking encapsulation), we can use the class constructor for the validation tests here right?
Also, add some tests for valid `Commits` as well.